spring-integration-core/src/main/java/org/springframework/integration/graph/IntegrationGraphServer.java

```diff
@@ -78,22 +79,22 @@ public class IntegrationGraphServer implements ApplicationContextAware, Applicat
 
 	private final NodeFactory nodeFactory = new NodeFactory(this::enhance);
 
-	private MicrometerNodeEnhancer micrometerEnhancer;
+	private @Nullable MicrometerNodeEnhancer micrometerEnhancer;
 
-	private ApplicationContext applicationContext;
+	private @Nullable ApplicationContext applicationContext;
```

Comment:
`@SuppressWarnings("NullAway.Init")` instead since this is set by the `ApplicationContextAware`